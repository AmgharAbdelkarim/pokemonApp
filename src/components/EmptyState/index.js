import React from 'react';

const EmptyState = () => {
  return <h3>No Data Found</h3>;
};

export default EmptyState;
