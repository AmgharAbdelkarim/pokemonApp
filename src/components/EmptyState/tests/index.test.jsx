import { cleanup, render } from "@testing-library/react";

import EmptyState from "components/EmptyState";

afterEach(cleanup);

describe("EmptyState", () => {
  it("should EmptyState render correctly", () => {
    const { asFragment, getByText } = render(EmptyState());
    expect(getByText("No Data Found")).toBeInTheDocument();
    expect(asFragment()).toMatchSnapshot();
  });
});
