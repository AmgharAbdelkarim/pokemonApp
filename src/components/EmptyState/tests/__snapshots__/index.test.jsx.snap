// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyState should EmptyState render correctly 1`] = `
<DocumentFragment>
  <h3>
    No Data Found
  </h3>
</DocumentFragment>
`;
